mix/qml/StatesComboBox.qml

```diff
@@ -0,0 +1,240 @@
+                    title: ""
+                    width: statesComboBox.width;
+                    delegate: mainItemDelegate
+                }
+                rowDelegate:  Rectangle {
+                    width:statesComboBox.width;
+                    height: statesComboBox.rowHeight;
+                }
+                Component {
+                    id: mainItemDelegate
+                    Rectangle {
+                        id: itemDelegate
+                        width:statesComboBox.width;
+                        height: statesComboBox.height;
+                        Text {
+                            id: textItemid
+                            text: styleData.value
+                            color: statesComboBox.colorItem;
+                            anchors.top: parent.top;
+                            anchors.left: parent.left;
+                            anchors.margins: 5;
+
+                        }
+                        Image {
+                            id: imageItemid
+                            height:20
+                            width:20;
+                            anchors.right:parent.right
+                            anchors.top: parent.top;
+                            anchors.margins: 5;
+                            visible: false;
+                            fillMode: Image.PreserveAspectFit
+                            source: "img/edit_combox.png"
+                        }
+
+                        MouseArea {
+                            anchors.fill: parent;
+                            hoverEnabled : true
+
+                            onEntered: {
+                                imageItemid.visible = true;
+                                textItemid.color = statesComboBox.colorSelect;
+                            }
+                            onExited: {
+                                imageItemid.visible = false;
+                                textItemid.color = statesComboBox.colorItem;
+                            }
+                            onClicked: {
+                                if (mouseX > imageItemid.x && mouseX < imageItemid.x+ imageItemid.width
+                                        && mouseY > imageItemid.y && mouseY < imageItemid.y+ imageItemid.height)
+                                    statesComboBox.editItem(styleData.row);
+                                else {
+                                    statesComboBox.state = ""
+                                    var prevSelection = chosenItemText.text
+                                    chosenItemText.text = styleData.value
+                                    listView.currentRow = styleData.row;
+                                    statesComboBox.selectItem(styleData.row);
+                                }
+                            }
+                        }
+                    }//Item
+                }//Component
+            }//Table View
+
+            RowLayout{
```

Comment:
RowLayout {

instead of

RowLayout{


2015-03-31 13:39 GMT+02:00 Ali Mashatan <notifications@github.com>:

> In mix/qml/StatesComboBox.qml
> <https://github.com/ethereum/cpp-ethereum/pull/1472#discussion_r27471432>:
>
> > +                                        && mouseY > imageItemid.y && mouseY < imageItemid.y+ imageItemid.height)
> > +                                    statesComboBox.editItem(styleData.row);
> > +                                else {
> > +                                    statesComboBox.state = ""
> > +                                    var prevSelection = chosenItemText.text
> > +                                    chosenItemText.text = styleData.value
> > +                                    listView.currentRow = styleData.row;
> > +                                    statesComboBox.selectItem(styleData.row);
> > +                                }
> > +                            }
> > +                        }
> > +                    }//Item
> > +                }//Component
> > +            }//Table View
> > +
> > +            RowLayout{
>
> I don't understand, could you example for me ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ethereum/cpp-ethereum/pull/1472/files#r27471432>.
>